plugins/prometheus_plugin/metrics.hpp

```diff
@@ -52,20 +52,42 @@ struct catalog_type {
    Gauge& last_irreversible;
    Gauge& head_block_num;
 
+   struct block_metrics {
+      Counter& num_blocks;
+      Gauge&   block_num;
```

Comment:
I find `block_num` and `num_blocks` confusing naming. On lines `124` and `125` there is explanation. I suggest to name them accordingly to be self-descriptive. Something like that or maybe you have better ideas:
`num_blocks` -> `produced_blocks_num`
`block_num` -> `cur_block_num`

Reply:
Renamed